buildSrc/src/main/java/Config.kt

```diff
@@ -72,6 +74,10 @@ object Config {
     }
 
     object QualityPlugins {
+        object Jacoco {
+            val version = "0.8.5"
+            val minimumCoverage = BigDecimal.valueOf(0.6)
+        }
         val jacocoVersion = "0.8.5"
```

Comment:
This should be deleted?

Reply:
Good catch. I'll remove it in Spring PR.